src/Nethermind/Nethermind.Core.Test/Crypto/BlsTests.cs

```diff
@@ -0,0 +1,77 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using FluentAssertions;
+using Nethermind.Crypto;
+using NUnit.Framework;
+
+namespace Nethermind.Core.Test.Crypto;
+
+using G1 = Bls.P1;
+using G2 = Bls.P2;
+using GT = Bls.PT;
+
+[TestFixture]
+public class BlsTests
+{
+    private BlsSigner.PrivateKey sk;
+
+    [SetUp]
+    public void Setup()
+    {
+        sk.Bytes = [0x2c, 0xd4, 0xba, 0x40, 0x6b, 0x52, 0x24, 0x59, 0xd5, 0x7a, 0x0b, 0xed, 0x51, 0xa3, 0x97, 0x43, 0x5c, 0x0b, 0xb1, 0x1d, 0xd5, 0xf3, 0xca, 0x11, 0x52, 0xb3, 0x69, 0x4b, 0xb9, 0x1d, 0x7c, 0x22];
+    }
```

Comment:
If this would be a method invoked in every test, we could parallelize the tests.